target/linux/ath79/dts/ar9342_ubnt_nanostation-loco-m-xw.dts

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9342_ubnt_xw.dtsi"
+
+/ {
+	compatible = "ubnt,nanostation-loco-m-xw", "ubnt,xw", "qca,ar9342";
+	model = "Ubiquiti Nanostation Loco M (XW)";
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy-mask = <0x1>;
+
+	phy4: ethernet-phy@4 {
```

Comment:
After what I've learned with the mikrotik PR, this should be
`phy1: ethernet-phy@1 {` ?

Reply:
It should. I haven't tested it on the real hardware but it's just a matter of naming, isn't it?

Reply:
I understand it the same way, it should be purely cosmetical.